### DISCUSSION TITLE
Sofar hyd 15ktl-3ph

### DISCUSSION THREAD

**Discussion by mariomurrent-softwaresolutions** [davidrapan/ha-solarman#349] [Q&A] [answered]
Which configuration should I use for the sofar hyd 15ktl-3ph?
I am currently using the auto setting, but the numbers are totally wrong.

**Comment by davidrapan** [comment, marked as answer]
Auto is just for Deye compatible devices.

You should proly use `sofar_g3hyd.yaml`.

**Reply by mariomurrent-softwaresolutions** [reply]
Ah damn must have missed this one. Thx, works like a charm.

**Comment by d4ystek** [comment]
hello
i have a problem with integration exactly every 2min and 10sec. entities become unavailable (integration disconnected) then after 20sec connected and so it repeats. Do you know what could be wrong? maybe some setting is wrong? The inverter is sofar ktlx5.5

**Reply by davidrapan** [reply]
What profile are you using?

**Reply by d4ystek** [reply]
Hello
 I am using the sofar_lsw_3 profile. The logger I have is LSE-3

**Reply by davidrapan** [reply]
Enable debug logging for few minutes and upload the file here. 😉
